packages/web/package.json

```diff
@@ -23,7 +23,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint . --ext .jsx,.js,.tsx,.ts",
```

Comment:
Try building the project (so that `build` directory appears) and run the lint command, you'll get something like:
```
✖ 31141 problems (30903 errors, 238 warnings)
  24122 errors and 0 warnings potentially fixable with the `--fix` option.
```

Guess we'd better limit linter to `./src` for now.

Reply:
done